examples/disable-canonical-redirect/gatsby-browser.js

```diff
@@ -0,0 +1,3 @@
+exports.onClientEntry = () => {
+  window.browserLoc.pathname = window.pagePath
```

Comment:
Ideally, this could be integrated in https://github.com/wardpeet/gatsby-plugin-static-site